File: src/LinksNews/Core/Enums.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace LinksNews.Core
{
    public enum ColumnTypes { Links = 1, News = 2, Rows = 3 };

    public enum RowAction { Unchanged, Inserted, Updated, Deleted};

    public enum PageElement { Page, Column, Row, Link};

    public enum ExceptionSeverity { Info = 1, Warning = 2, Error = 3 }

    public enum MessageGroups { Site = 1, ContactUs = 2, Forum = 3 }

    public enum ViewModes { List = 1, Tile = 2 }

    public enum NewsProviderTypes { Api = 1, Rss = 2 }
}

File: src/LinksNews/angular2App/app/core/common/enums.ts
export enum DialogButtonType {
    OK,
    Yes,
    No,
    Cancel,
    Close
}

export enum ColumnTypes {
    Links = 1, 
    News = 2,
    Rows = 3 
}


export enum RowAction { 
    Unchanged, 
    Inserted, 
    Updated, 
    Deleted
};


export enum PageElement { 
    Page, 
    Column, 
    Row, 
    Link
};

export enum ViewModes {
    List = 1, 
    Tile = 2
}

export enum SortOrder {
    None,
    Asc,
    Desc
}
